app/repositories/PlaybackRepository.scala

```diff
@@ -16,92 +16,67 @@
 
 package repositories
 
-import play.api.Configuration
-import play.api.libs.json._
-import play.modules.reactivemongo.ReactiveMongoApi
-import reactivemongo.api.WriteConcern
-import reactivemongo.api.bson.BSONDocument
-import reactivemongo.api.bson.collection.BSONSerializationPack
-import reactivemongo.api.indexes.Index.Aux
-import reactivemongo.api.indexes.{Index, IndexType}
-import reactivemongo.play.json.collection.Helpers.idWrites
+import com.mongodb.client.model.Indexes.ascending
+import com.mongodb.client.model.ReturnDocument
+import config.AppConfig
 import models.UserAnswers
+import org.mongodb.scala.bson.conversions.Bson
+import org.mongodb.scala.model._
+import uk.gov.hmrc.mongo.MongoComponent
+import uk.gov.hmrc.mongo.play.json.PlayMongoRepository
 
 import java.time.LocalDateTime
 import javax.inject.{Inject, Singleton}
+import scala.concurrent.duration.SECONDS
 import scala.concurrent.{ExecutionContext, Future}
 
 @Singleton
-class PlaybackRepositoryImpl @Inject()(
-                                        override val mongo: ReactiveMongoApi,
-                                        override val config: Configuration
-                                      )(override implicit val ec: ExecutionContext)
-  extends IndexesManager with PlaybackRepository {
-
-  override val collectionName: String = "user-answers"
-
-  override val cacheTtl: Int = config.get[Int]("mongodb.playback.ttlSeconds")
-
-  override val lastUpdatedIndexName: String = "user-answers-updated-at-index"
-
-  override def idIndex: Aux[BSONSerializationPack.type] = Index.apply(BSONSerializationPack)(
-    key = Seq("newId" -> IndexType.Ascending),
-    name = Some("internal-id-and-utr-and-sessionId-compound-index"),
-    expireAfterSeconds = None,
-    options = BSONDocument.empty,
-    unique = false,
-    background = false,
-    dropDups = false,
-    sparse = false,
-    version = None,
-    partialFilter = None,
-    storageEngine = None,
-    weights = None,
-    defaultLanguage = None,
-    languageOverride = None,
-    textIndexVersion = None,
-    sphereIndexVersion = None,
-    bits = None,
-    min = None,
-    max = None,
-    bucketSize = None,
-    collation = None,
-    wildcardProjection = None
-  )
-
-  private def selector(internalId: String, identifier: String, sessionId: String): JsObject = Json.obj(
-    "newId" -> s"$internalId-$identifier-$sessionId"
-  )
-
-  override def get(internalId: String, identifier: String, sessionId: String): Future[Option[UserAnswers]] = {
-    findCollectionAndUpdate[UserAnswers](selector(internalId, identifier, sessionId))
+class PlaybackRepository @Inject()(
```

Comment:
this class doesn't seem to be tested anywhere. You could add a spec for PlaybackRepository

Reply:
agreed that this will be done as part of test clean up tickets in the future